setup/updating.md

```diff
@@ -65,8 +65,47 @@ To update Coder, follow these steps:
    ```console
    helm upgrade --namespace coder --install --atomic --wait \
      --version 1.16.1 coder coder/coder --values current-values.yaml
+
    ```
 
+1. [Update your Helm chart](../guides/admin/helm-charts.md) via the steps below.
+Refer to the `1.21` [helm chart](https://github.com/cdr/enterprise-helm/blob/main/values.yaml)
+when making such changes:
+
+   Rename all instances of `cemanager` to `coderd`.
+
+   Remove all `dashboard.*` values. Route any traffic to `coderd`.
+
+   Remove all `envproxy.*` values. Route any traffic to `coderd`.
+
+   Redirect all traffic from your custom ingress to `coderd`.
+
+   `coderd` now serves TLS certificates; to upgrade the NGINX ingress
+   controller, set `coderd.serviceNext` to `true`. (See our
+   [TLS certificates](../guides/tls-certificates/index.md) for more information
+   on how to configure Coder to issue and use certificates.)
+
+1. Migrate the following ports. This may need to be done within your cloud provider:
```

Comment:
This is for custom ingresses right? The LoadBalancer should still pick up the service fine.

Reply:
This I'm not sure.

Reply:
Yes, custom ingresses